toptal/picasso
# chore(autocomplete): [PRO-1837] add the "powered by google" label

[PRO-1837]

### Description
There are situations when we use options in the Autocomplete component from the Google API and to meet their requirements https://developers.google.com/places/web-service/policies in such cases we have to show the "Powered By Google" label. 
After a discussion with @borisyordanov, we decided that the best option is to add this label right into Picasso.

### How to test
- Go to the storybook for the Autocomplete component;
- Scroll down till the new "Powered By Google label" section;
- Take a look at the new image at the end of the options list;
- Also it should not show itself if the list of the options is empty;

### Screenshots

![Screenshot 2020-11-05 at 21 26 17](https://user-images.githubusercontent.com/5062734/98281258-92585d80-1fad-11eb-9075-5062af23d2a0.png)

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>


[PRO-1837]: https://toptal-core.atlassian.net/browse/PRO-1837
